wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,303 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+# osu! Rankings History
+
+The following is a list of players who are recognized to have reached Rank #1 in the [osu! Global Rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (July 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – Present)
+
+Because of this, this list will only include players who reached Rank #1 on the main leaderboard that was used at the time of reaching said rank. *Italicized* dates and numbers indicate that the corresponding player are continuing their reign as osu!'s Rank #1 player either from the previous year or onto the following year.
+
+## 2007
+
+*Note: During the period where Ranked Score was used, Rank #1 would often change as of such this list only includes users who have been verified to have reached Rank #1. Early dates use months where exact dates aren't available.*
```

Comment:
```suggestion
*Note: During the period where Ranked Score was used, Rank #1 would change frequently. Becaues of this, this list will only includes user who have been verified to have reached Rank #1. Earlier listings may use months where exact dates aren't available.*
```
Sounds weird. Also, not sure why we need to have the "verified" bit because, isn't every listing verified here? If not, consider just removing it
```suggestion
*Note: During the period where Ranked Score was used, Rank #1 would change frequently, thus the earlier listings may use months where exact dates aren't available.*
```